add-on/chrome/modules/LoopSidebar.jsm

```diff
@@ -47,8 +54,10 @@ var LoopSidebarInternal = {
     sidebarBrowser.setAttribute("frameType", "loop");
     sidebarBrowser.setAttribute("messagemanagergroup", "loop");
     sidebarBrowser.setAttribute("message", "true");
-    sidebarBrowser.setAttribute("remote", "true");
-
+    // If the browser is in non-e10s mode, but we set remote=true, then the sidebar
+    // would run in remote mode regardless.
+    sidebarBrowser.setAttribute("remote", window.gMultiProcessBrowser ? "true" : "false");
+    sidebarBrowser.setAttribute("tooltip", "aHTMLTooltip");
```

Comment:
While the comment is true, it's a little obtuse.  Perhaps phrase it as "ensure that the sidebar only runs in the remote process if e10s is turned on"?